New version: Unity.Unity.2022 version 2022.3.13f1

Dragon1573 opened the pull request:
## Description

- Manifests created via [Komac v1.11.0](https://github.com/russellbanks/Komac/releases/tag/v1.11.0) on [GitHub Codespace](https://github.com/codespaces)
- Resolve https://github.com/microsoft/winget-pkgs/issues/127093

## Checks

- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] This PR only modifies one (1) manifest
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [ ] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.5 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.5.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

---

 ###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/microsoft/winget-pkgs/pull/127308)

Dragon1573 committed:
New version Unity.Unity.2022 version 2022.3.13f1

Signed-off-by: GitHub <noreply@github.com>

stephengillie commented:
Hi @Dragon1573,

Manual Validation ended with:
![image](https://github.com/microsoft/winget-pkgs/assets/114952565/ed568b23-5bec-4ac6-8956-2afd39100a00)

Should this package have `ElevationRequirement: elevationProhibited`?

Dragon1573 committed:
Update (installer.yaml): ElevationRequirement

- Unity is not designed to run as Administrator, elevation behaviour should be prohibited.

Signed-off-by: Dragon1573 <49941141+Dragon1573@users.noreply.github.com>

Dragon1573 commented:
@stephengillie - According to #102996 , you labeled this property with `ElevationRequirement: elevatesSelf`. But that PR can successfully pass the pipeline.

Wierd ...

Dragon1573 committed:
Revert "Update (installer.yaml): ElevationRequirement"

This reverts commit 78bedf37828ac28c2a96fcc89b503c50f9870379.

stephengillie commented:
> @ stephengillie - According to #102996 , you labeled this property with `ElevationRequirement: elevatesSelf`. But that PR can successfully pass the pipeline.
> 
> Wierd ...

Hi @Dragon1573,

This package passes the basic test of "launch and remain running for 10 seconds, then exit without errors" when running as administrator, while displaying a message that this is a bad idea.
